src/app/components/AssignmentList.tsx

```diff
@@ -12,18 +12,18 @@ export interface AssignmentListProps {
 }
 
 class AssignmentList extends  React.PureComponent<AssignmentListProps, any>{
-    render() {
```

Comment:
Can we get rid of this component since it doesn't appear to be doing anything anymore?